.github/workflows/sdk-jobs-automl-standalone-jobs-automl-image-classification-multiclass-task-fridge-items-automl-image-classification-multiclass-task-fridge-items.yml

```diff
@@ -25,6 +25,8 @@ jobs:
         python-version: "3.8"
     - name: pip install notebook reqs
       run: pip install -r sdk/dev-requirements.txt
+    - name: pip install mkflow reqs
+      run: pip install -r sdk/mlflow-requirements.txt
```

Comment:
[Typo] mkflow -> mlflow.
Applicable to other workflow yamls also

Reply:
its an auto generated file. fixed source script.